### ISSUE TITLE
Check target attribute in links

### ISSUE THREAD

**Issue by taitus** [consuldemocracy/consuldemocracy#4355]
### Description
*Type*: Bug

### Current Behavior
When I click a link which mentions it opens in a new window, it does not open in a new window.

### Expected Behavior
When I click a link which mentions it opens in a new window, it opens in a new window.

### Steps to reproduce
1. Go to sign_up page
2. Click on `terms and conditions of use`

**Comment by Isaius** [comment]
By that you mean a new tab?
You're right. It really says that but opens in the same window.
In [this line](https://github.com/consul/consul/blob/63820fb59fa4f9af0d1d6f13b50a9ec2756f2b02/app/views/users/registrations/new.html.erb#L44) the target is set to "_blank", but the page doesn't contain that.

![image](https://user-images.githubusercontent.com/37005228/108117529-c696ec00-707b-11eb-9abd-bd580d5176e2.png)

**Comment by javierm** [comment]
> By that you mean a new tab?

@Isaius Yes :wink:.

**Comment by Isaius** [comment]
I'll try to find it out why isn't working.

**Comment by Isaius** [comment]
I've found the cause. The ActionView Sanitizer helper isn't allowing the target tag. As you can see [here](https://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html)

Just adding this to the `application.rb` file solves the problem.

```rb
# In config/application.rb
config.action_view.sanitized_allowed_tags = ['strong', 'em', 'a']
config.action_view.sanitized_allowed_attributes = ['href', 'title']
```

**Comment by Isaius** [comment]
Reading the #4223 issue I think that's intentional, @javierm ?

**Comment by javierm** [comment]
@Isaius It's something that might need debate in the future :thinking:. In the meantime, we're trying to keep the behavior we had in version 1.0.

Regarding this issue, maybe we could allow the `target` attribute when [sanitizing check box labels](https://github.com/consul/consul/blob/master/lib/consul_form_builder.rb#L24) :thinking:. Do you think it sounds reasonable, at least for now?

**Comment by Isaius** [comment]
I think so. Open the terms in a new window makes more sense to me, since stop the registration to read them, then comeback to the page may lead the user to close that tab instead of backwards one page, losing the already inserted data.

**Comment by Isaius** [comment]
@javierm You mean adding `target` to [this line](https://github.com/consul/consul/blob/2490d5e2093e682f31263fe4ff8a9514ff6d40a2/lib/wysiwyg_sanitizer.rb#L7)? 
If so, tried that and not worked.

**Comment by javierm** [comment]
I was wondering whether it would be fine to add an `attributes` param in the call to `sanitize` when [sanitizing check box labels](https://github.com/consul/consul/blob/master/lib/consul_form_builder.rb#L24) in order to include the `target` attribute.

**Comment by DiegoZen** [comment]
Is this still happening in every sanitized text. I solve it adding this line in `application.rb`:
```rb
config.after_initialize do
      Globalize.set_fallbacks_to_all_available_locales
      # new line
      ActionView::Base.sanitized_allowed_attributes.add 'target' 
end
```

May I open a pull request for this?

**Comment by javierm** [comment]
Hi, @DiegoZen :smile:.

For now, as mentioned earlier, I'd rather add an `attributes` param in the call to `sanitize` when [sanitizing check box labels](https://github.com/consul/consul/blob/master/lib/consul_form_builder.rb#L26) in order to include the `target` attribute.

The reason is that automatically opening links in a new tab is an accessibility/usability issue (and removing some of the existing `target` attributes is on our TODO list), and allowing it globally would encourage this practice.

**Comment by javierm** [comment]
Closed via #5283.
